diesel_cli/src/database.rs

```diff
@@ -303,19 +320,22 @@ mod tests {
 
     #[test]
     fn split_pg_connection_string_returns_postgres_url_and_database() {
```

Comment:
As this is also used for mysql connections, can you add a test with examples for that as well?